common/ledger/blockledger/fileledger/factory.go

```diff
@@ -90,8 +101,28 @@ func New(directory string, metricsProvider metrics.Provider) (blockledger.Factor
 	if err != nil {
 		return nil, err
 	}
+
+	fileRepo, err := filerepo.New(directory, "remove")
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This deviates from the pattern we already have in here: 

https://github.com/hyperledger/fabric/blob/9adde2e7d30e1cb8778f8dbb564e08bc49afb70d/orderer/common/multichannel/registrar.go#L120

where we put all different file-repos under the ledger directory in a "filerepo" folder. The "remove" will be another subfolder, along side the "joinblock" file-repo. 
This way we'll have 
```
- ledgerDir
    - chains
        - mychain
        - yourchain
    - filerepo
        - joinblock
        - remove
```

Reply:
🧹